### PR TITLE
Command.sga tests

### PR DESCRIPTION
**Pull request by Jameskmonger**
This adds full test coverage around the `sga` command.

### CODE REVIEW

File: tests/commands/sgaTests.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+var sga = require("__buttercup/classes/commands/command.sga.js");
+
+var fakeGroup;
+
+module.exports = {
+  setUp: function(cb) {
+    this.command = new sga();
+    (cb)();
```
Comment (perry-mitchell) [inline tests/commands/sgaTests.js]: Indenting :)

Comment (Jameskmonger) [inline tests/commands/sgaTests.js]: Ah I've figured it out - Atom on Windows for me was set to only indent 2 spaces. Atom on Ubuntu was set to 4 spaces. I'll fix this in about 25 minutes.

Comment (Jameskmonger) [inline tests/commands/sgaTests.js]: Odd that `npm test` didn't pick this up though...

Comment (perry-mitchell) [inline tests/commands/sgaTests.js]: Only have eslint running source. Tried on tests and it threw hundreds of errors - will make that a task :)

**Comment by Jameskmonger** [comment]
Fixed the indenting @perry-mitchell  - ready for merge :smile:

**Comment by perry-mitchell** [comment]
Looks like sgaTests is still indented at 2, whilst cgr looks good..

**Comment by Jameskmonger** [comment]
Not sure what happened there, Atom for some reason only fixed the indenting in cgr... Give me a second

**Comment by Jameskmonger** [comment]
Indentation fixed in sga tests @perry-mitchell

**Pull request merged**